pkgs/applications/audio/x42-mclk/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, lv2 }:
```

Personally, I recommend:

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, pkg-config
, lv2
}:
```